### ISSUE TITLE
Restore script/process

### ISSUE THREAD

**Issue by ravensorb** [pothi/backup-wordpress#22]
I was curious if you had a recommended approach or script for a restore?

**Comment by pothi** [comment]
Good idea. However, since the restoration may depend on the environment (such as the DB name, password, etc), there are certain difficulties in it. Probably, with an `.envrc` file, the restoration process could be streamlined. I will keep this open for further discussion or ideas. Thanks.
